src/batch-client/index.ts

```diff
@@ -1045,7 +1045,16 @@ export class ZimbraBatchClient {
 				offset
 			},
 			namespace: Namespace.Account
-		}).then(res => normalize(DlGroupMember)(get(res, 'groupMembers.0.groupMember') || []));
+		}).then(res => {
+			if (res?.dlm) {
+				return res;
+			} else {
+				const result = {
+					dlGroupMember: normalize(DlGroupMember)(get(res, 'groupMembers.0.groupMember') || [])
```

Comment:
we don't prefer using lodash#get anymore

Reply:
@silentsakky Done.